gobblin-api/src/main/java/org/apache/gobblin/service/ServiceConfigKeys.java

```diff
@@ -35,6 +35,7 @@
   public static final String GOBBLIN_SERVICE_GIT_CONFIG_MONITOR_ENABLED_KEY = GOBBLIN_SERVICE_PREFIX + "gitConfigMonitor.enabled";
   public static final String GOBBLIN_SERVICE_DAG_MANAGER_ENABLED_KEY = GOBBLIN_SERVICE_PREFIX + "dagManager.enabled";
   public static final String GOBBLIN_SERVICE_JOB_STATUS_MONITOR_ENABLED_KEY = GOBBLIN_SERVICE_PREFIX + "jobStatusMonitor.enabled";
+  public static final String GOBBLIN_SERVICE_DELAYED_START = GOBBLIN_SERVICE_PREFIX + "d2.delayedStart";
```

Comment:
Can we name this after intention, rather than implementation? Looks like what we want to configure here is "send d2 requests only to master node". You can also add a comment when this is needed - if d2 randomly balances requests across all servers, some of the write requests will fail.

Reply:
Changed to `d2.onlyAnnounceLeader`